src/dictionary/dictionary.c

```diff
@@ -495,3 +495,44 @@ dictionary_find(
 ) {
 	return dictionary->handler->find(dictionary, predicate, cursor);
 }
+
+ion_boolean_t
+test_predicate(
+	ion_dict_cursor_t		*cursor,
+	ion_key_t				key,
+	ion_dictionary_parent_t parent
```

Comment:
Lets re-order these parameters to be parent, cursor, key. More in line with the rest of the API.